File: kotlin_java/src/main/java/baekjoon/bronze/b1/수_정렬하기_3.java
package baekjoon.bronze.b1;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class 수_정렬하기_3 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int inputCase = Integer.parseInt(br.readLine());
        int[] arr = new int[10001];
        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < inputCase; i++) {
            arr[Integer.parseInt(br.readLine())]++;
        }

        for (int i = 0; i < 10001; i++) {
            for (int j = 0; j < arr[i]; j++) {
                sb.append(i).append("\n");
            }
        }

        System.out.println(sb);
    }
}

File: kotlin-java/src/main/kotlin/baekjoon/bronze/b1/수 정렬하기 3.kt
package baekjoon.bronze.b1

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val case = br.readLine().toInt()
    val arr = Array(10001) { 0 }

    for (i in 0..<case) {
        arr[br.readLine().toInt()]++
    }

    br.close()

    val sb = StringBuilder()

    for (i in 1..<10001) {
        for (j in 0..<arr[i])
            sb.append(i).append("\n")
    }
    print(sb.toString())
}